data_ingestion/common/primary_care_access.py

```diff
@@ -0,0 +1,68 @@
+from .di_url_file_to_gcs import try_urls_to_download_file_to_gcs
+import logging
+import os
+from google.cloud import storage
+import google.cloud.exceptions
+import requests
+
+STATE_NAMES = [
+"Alabama",
```

Comment:
nit: +2 or +4 indent (idk which is more correct), and put the last bracket on its own line